src/Illuminate/Validation/Rules/Base64.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Illuminate\Validation\Rules;
+
+use Illuminate\Contracts\Validation\Rule;
+
+class Base64 implements Rule
+{
+
+    /**
+     * Determine if the validation rule passes.
+     *
+     * @param  string  $attribute
+     * @param  mixed  $value
+     * @return bool
+     */
+    public function passes($attribute, $value)
+    {
+        $result = base64_decode($value, true);
+
+        if($result === false) {
+            return $result;
+        }
+
+        return true;
```

Comment:
Can't this be reduced to `return base64_decode($value, true) !== false;`?

Reply:
Can it will be there now ;-)